guzzlehttp/guzzle/CVE-2016-5385.yaml

```diff
@@ -0,0 +1,14 @@
+title:     HTTP Proxy header vulnerability
+link:      https://github.com/guzzle/guzzle/releases/tag/6.2.1
+cve:       CVE-2016-5385
+branches:
+    master:
+        time:     2015-07-15 17:14:23
+        versions: ['>=6 <6.2.1']
```

Comment:
Thanks again. Next time, I promise to do a better job of reading [the contributors documentation](https://github.com/FriendsOfPHP/security-advisories#contributing). 😳